doc/paper.md

```diff
@@ -42,37 +42,38 @@ related forecasting applications in research and production HPC environments,
 and it is now part of the official software infrastructure for the Unified
 Model atmospheric model. Cylc is written in Python and developed primarily by
 NIWA (NZ) and Met Office (UK). It has strong support for large production
-systems but is not in any way specialized to environmental forecasting, and
-ease of use for individuals with smaller workflow automation requirements
-remains a key priority.
+systems, but ease of use for individuals with smaller workflow automation
+requirements remains a key priority, and despite its core user base it is not
+in any way specialized to environmental forecasting.
 
 In cycling workflows tasks repeat on sequences that may represent forecast
-cycles, or successive chunks of a simulation that is too long for a single run,
-or steps in some multi-program iterative process (e.g. for optimizing model
-parameters), or successive datasets to be processed as they are generated or
-received, and so forth. Cycling in Cylc is controlled by ISO 8601 date-time
-recurrence expressions (e.g. for environmental forecasting), or integer
-recurrence expressions (e.g. for iterative processes). Dependence across cycles
-creates ongoing, potentially never-ending, workflows. Cylc can uniquely manage
-these without imposing a global cycle loop: one cycle does not have to complete
-before the next can start. Instead, Cylc's novel meta-scheduling algorithm runs
-tasks from many cycles at once, to the full extent allowed by individual
-dependencies and external constraints such as compute resource and data
-availability. So, for example, on restarting after extended downtime, a
-workflow that processes real-time data can clear its backlog and catch up very
-quickly by interleaving cycles.
+cycles, chunks of a simulation that is too long for a single run, steps in some
+multi-program iterative process (e.g. for optimizing model parameters), or
+datasets to be processed as they are generated or received, and so forth.
+Cycling in Cylc is controlled by ISO 8601 date-time recurrence expressions
+(e.g. for environmental forecasting), or integer recurrence expressions (e.g.
+for iterative processes). Dependence across cycles creates ongoing, potentially
+never-ending, workflows (rather than simply a succession of disconnected single
+workflows). Cylc can uniquely manage these workflows without imposing a global
+cycle loop: one cycle does not have to complete before the next can start.
+Instead, Cylc's novel meta-scheduling algorithm runs tasks from many cycles at
+once, to the full extent allowed by individual dependencies and external
+constraints such as compute resource and data availability. So, for example, on
```

Comment:
This implies that we support resource and data aware scheduling. Perhaps drop this line:

many cycles at once ~, to the full extent allowed by individual dependencies and external
+constraints such as compute resource and data availability~.

Reply:
My meaning was, cycle interleaving is (or can be) constrained by "external" delays, e.g. because jobs stuck in a PBS queue will naturally hold things up.  But I see what you mean, my statement could be misconstrued as implying that Cylc has (and uses) knowledge of external resource.